src/sailjail.c

```diff
@@ -369,7 +408,12 @@ int main(int argc, char* argv[])
             argc--;
         }
         if (argc > 1) {
-            ret = sailjail_main(argc, argv, conf, &args);
+            /* PROGRAM must be defined with an absolute path */
+            if (argv[1][0] != '/') {
```

Comment:
I agree. I was thinking the same while writing this (and also when writing the previous PR I think). I think the idea with giving all of argv to sailjail_main() is that it's main function so it should also get the zeroeth argument but at the same time it doesn't need it for anything. Similarly sailjail_sandbox() doesn't use it and I would argue that there the meaning of argv has changed so that there is no point passing that zeroeth value. Maybe that could be improved somehow.